app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,17 @@
+from .elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self,
```

Comment:
After the opening bracket, write all arguments on a new line, including the first one; also write the closing bracket on a new line